packages/core/src/types.ts

```diff
@@ -134,7 +136,7 @@ interface FetchMeta<
   resolve: (value?: any | PromiseLike<any>) => void;
   reject: (reason?: any) => void;
   promise: PromiseLike<any>;
-  createdAt: Date;
+  createdAt: number;
```

Comment:
also maybe this?

Reply:
made it number | Date, added back compat handling